ERCS/erc-7540.md

```diff
@@ -36,8 +36,8 @@ The existing definitions from [ERC-4626](./eip-4626.md) apply. In addition, this
 - Request: a request to enter (`requestDeposit`) or exit (`requestRedeem`) the Vault
 - Pending: the state where a Request has been made but is not yet Claimable
 - Claimable: the state where a Request is processed by the Vault enabling the user to claim corresponding `shares` (for async deposit) or `assets` (for async redeem)
-- Claimed: the state where a Request is finalized by the user and the user receives the output token (e.g. `shares` for a deposit Request)
-- Claim function: the corresponding Vault method to bring a Request to Claimed state (e.g. `deposit` or `mint` claims `shares` from `requestDeposit`). Lowercase claim always describes the verb action of calling a Claim function.
+- Claimed: the state where a Request is finalized by the user and the user receives the output token (i.e. `shares` for async deposit and `assets` for async redeem)
```

Comment:
I think the old phrasing here is fine, it is giving the example for the deposit Request.